CHANGELOG.md

```diff
@@ -3514,6 +3514,12 @@ This is a full list of proofs that have changed form to use irrelevant instance
 * Added new proof to `Induction.WellFounded`
   ```agda
   Acc-resp-flip-≈ : _<_ Respectsʳ (flip _≈_) → (Acc _<_) Respects _≈_
+
+  acc-asym : ∀ {x y} → Acc _<_ x → x < y → ¬ (y < x)
```

Comment:
Style guide advises removing the initial `∀ {x y} →` prefix here, as it can be inferred by the reader.

Reply:
Fixed as suggested